hydra-node/test/Hydra/OptionsSpec.hs

```diff
@@ -26,6 +29,21 @@ spec = parallel $
   describe "Hydra Node RunOptions" $ do
     -- NOTE: --node-id flag needs to be set so we set a default here
     let setFlags a = ["--node-id", "node-id-1"] <> a
+    it ("validateRunOptions: using more than " <> show (maximumNumberOfParties + 1) <> " parties should error out") $ do
+      let simulatedKeys = replicate 10 ['a' .. 'z']
```

Comment:
Aren't these always the same keys? i.e. `["abcd..z", "abcd...z", ... ]` with 10 such entries.

I would suggest to generate some random values here. Maybe with something like:

```
  generateKeyString = vectorOf 10 $ chooseEnum ('a', 'z')
  cardanoKeys = replicateM (maxParties + 1) generateKeyString `generateWith` 42 -- this is a seed
```